packages/api/orm/entity/Portfolio.ts

```diff
@@ -0,0 +1,32 @@
+import { Application } from "./Application";
+import { CloudServiceProvider } from "../../models/CloudServiceProvider";
+import { Column, Entity, OneToMany } from "typeorm";
+import { ProvisionableEntity } from "./ProvisionableEntity";
+import { TaskOrder } from "./TaskOrder";
+
+@Entity()
+export class Portfolio extends ProvisionableEntity {
+  @Column({ type: String, length: 100 })
+  name: string;
+
+  @Column({ type: String, nullable: true, length: 300 })
+  description: string;
+
+  @Column({ type: String })
+  owner: string;
```

Comment:
I noticed you removed the type in the `column` for string, should these be removed as well?

Reply:
Yes.  Nice catch.

Reply:
Updated in 06b254a386c831c08e7d400788fa2bf6d17fc6a7